{
	"id": "64094",
	"key": "TIMOB-3462",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "10153",
			"key": "TIMOB",
			"name": "Titanium SDK/CLI",
			"projectCategory": {
				"id": "10100",
				"description": "Titanium and related SDKs used in application development",
				"name": "Client"
			}
		},
		"fixVersions": [],
		"resolution": {
			"id": "5",
			"description": "All attempts at reproducing this issue failed, or not enough information was available to reproduce the issue. Reading the code produces no clues as to why this behavior would occur. If more information appears later, please reopen the issue.",
			"name": "Cannot Reproduce"
		},
		"resolutiondate": "2016-08-19T21:41:01.000+0000",
		"created": "2011-04-15T03:45:30.000+0000",
		"priority": {
			"name": "Low",
			"id": "4"
		},
		"labels": [],
		"versions": [],
		"issuelinks": [
			{
				"id": "52491",
				"type": {
					"id": "10002",
					"name": "Duplicate",
					"inward": "is duplicated by",
					"outward": "duplicates"
				},
				"inwardIssue": {
					"id": "64095",
					"key": "TIMOB-3463",
					"fields": {
						"summary": "Can't have a \"é\" in my app name...",
						"status": {
							"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
							"name": "Closed",
							"id": "6",
							"statusCategory": {
								"id": 3,
								"key": "done",
								"colorName": "green",
								"name": "Done"
							}
						},
						"priority": {
							"name": "Low",
							"id": "4"
						},
						"issuetype": {
							"id": "1",
							"description": "A problem which impairs or prevents the functions of the product.",
							"name": "Bug",
							"subtask": false
						}
					}
				}
			}
		],
		"assignee": {
			"name": "ingo",
			"key": "ingo",
			"displayName": "Ingo Muschenetz",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"updated": "2017-03-23T20:23:52.000+0000",
		"status": {
			"description": "The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.",
			"name": "Closed",
			"id": "6",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "10206",
				"name": "iOS",
				"description": "iOS Platform"
			}
		],
		"description": "Any compile is ok,but when my app's name contains \"&eacute;\", there is an error:\r\n\r\n{code}\r\nInstall Error\r\nError: Traceback (most recent call last): File\r\n\"/Library/Application Support/Titanium/mobilesdk/osx/1.6.1/iphone/builder.py\", line 850, in main\r\nos.system(\"/usr/bin/plutil -convert binary1 \\\"%s\\\"\" % app_stylesheet)\r\nUnicodeEncodeError: 'ascii' codec can't encode character u'\\u0301' in position 69: ordinal not in range(128)\r\n{code}\r\n\r\nHow can I fix that?\r\nThanks;\r\nIvan.",
		"attachment": [],
		"flagged": false,
		"summary": "Can't have a \"é\" in my app name...",
		"creator": {
			"name": "ivanmathy",
			"key": "ivanmathy",
			"displayName": "Ivan Mathy",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"subtasks": [],
		"reporter": {
			"name": "ivanmathy",
			"key": "ivanmathy",
			"displayName": "Ivan Mathy",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"environment": null,
		"comment": {
			"comments": [
				{
					"id": "393673",
					"author": {
						"name": "cbarber",
						"key": "cbarber",
						"displayName": "Chris Barber",
						"active": true,
						"timeZone": "America/Chicago"
					},
					"body": "This is no longer an issue as of Titanium SDK 6 and probably fixed in Titanium SDK 3 or 4 era.",
					"updateAuthor": {
						"name": "cbarber",
						"key": "cbarber",
						"displayName": "Chris Barber",
						"active": true,
						"timeZone": "America/Chicago"
					},
					"created": "2016-08-19T21:41:01.000+0000",
					"updated": "2016-08-19T21:41:01.000+0000"
				},
				{
					"id": "415354",
					"author": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Closing ticket as \"Cannot Reproduce\" with reference to the above comments.",
					"updateAuthor": {
						"name": "lmorris",
						"key": "lmorris",
						"displayName": "Lee Morris",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2017-03-23T20:23:52.000+0000",
					"updated": "2017-03-23T20:23:52.000+0000"
				}
			],
			"maxResults": 2,
			"total": 2,
			"startAt": 0
		}
	}
}